app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_for @post do |f| %>
+
+<% if @post.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@post.errors.count, "error") %> prohibited
+      this comment from being saved:
+    </h2>
+    <ul>
+      <% @post.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<p>
+<%= f.label :text %>
+<%= f.text_field :text %>
+</p>
+
+<p>
+<%= f.label :commenter %>
+<%= f.text_field :commenter %>
+</p>
+
+<%= f.submit %>
+
+<% end %>
```

Comment:
was wondering if this worked. It's a form for posts but looks like commenter fields.